### ISSUE TITLE
Implement the Cypress e2e test to test the login flow of krini

### ISSUE THREAD

**Issue by git-anurag-hub** [elixir-cloud-aai/krini#1]
**Is your feature request related to a problem? Please describe.**
As the CI pipelirunsrun certain tests on the Krini before deploying it to the production. Checking the login flow will athe good test to be written in the case OIDC auth is broken by any code changes.

**Describe the solution you'd like**
We can write the cypress e2e test that test for the login flow before deploying the application to production.

**Describe alternatives you've considered**
No alternatives.

**Additional context**
Test can be written in the file `./cypress/e2e/spec.cy.js` under the test `auth -> user can login`. This test is already integrated in the CI pipeline & this can be tested by running `npm run cypress:open`.

**Comment by tanya17-05** [comment]
Hi @git-anurag-hub, can you please assign this issue to me?

**Comment by uniqueg** [comment]
Hi @tanya17-05, I think @Sanyam2000-dot is already working on this, but I'm not sure. @git-anurag-hub: can you please write some more issues and make sure both @Sanyam2000-dot and @tanya17-05 have issues to work on? Thanks 🙏🏻

**Comment by git-anurag-hub** [comment]
Yeah sure @uniqueg. @Sanyam2000-dot can you confirm if you are working on this?

**Comment by Sanyam2000-dot** [comment]
Hello Anurag, Yes I'm working on this issue. Thanks!

**Comment by Sanyam2000-dot** [comment]
> Ok @Sanyam2000-dot, let me know if you need any kind of help.

Thanks @tanya17-05 . I'll ket you know.

**Comment by git-anurag-hub** [comment]
Hey @Sanyam2000-dot are you working on this?

**Comment by Sanyam2000-dot** [comment]
Hello @git-anurag-hub, I have started this but have not completed it due to some health issues in the family. It will be good if you can assign it to Tanya so I can guide her forward.

**Comment by git-anurag-hub** [comment]
Np @Sanyam2000-dot , TC!

**Comment by SohamRatnaparkhi** [comment]
Hi,
Can I work on this issue. I have some experience in writing cyprus tests.
My approach would be, using cyprus:
 - clicking on login button on home screen
 - clicking on `Login with LS username"
 - entering test username and password
 - clicking on login button
 - searching for `Run workflow` button to confirm login was successful. 
 
 I am open for suggestions to my approach.

**Comment by git-anurag-hub** [comment]
Sure @SohamRatnaparkhi, go ahead with it!

**Comment by git-anurag-hub** [comment]
Hey @SohamRatnaparkhi, you can right now try out with yours & configure the same in the .env.test!
